jellyfin-core/src/main/kotlin/org/jellyfin/apiclient/discovery/RecommendedServerInfoScore.kt

```diff
@@ -0,0 +1,9 @@
+package org.jellyfin.apiclient.discovery
+
+enum class RecommendedServerInfoScore(
+	val i: Int
```

Comment:
Even if repeated, it's way more insightful that way. Inline classes would be the perfect match here, but I don't know whether they're still marked as "unstable" or not..
```suggestion
	val score: Int
```

Reply:
Forgot to change it